[2025-10-28T23:43:01.182+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-28T23:42:00+00:00 [queued]>
[2025-10-28T23:43:01.195+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-28T23:42:00+00:00 [queued]>
[2025-10-28T23:43:01.196+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-10-28T23:43:01.215+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): ConsumKafka> on 2025-10-28 23:42:00+00:00
[2025-10-28T23:43:01.225+0000] {standard_task_runner.py:60} INFO - Started process 174 to run task
[2025-10-28T23:43:01.230+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag1_calcul_trajet_kafka', 'ConsumKafka', 'scheduled__2025-10-28T23:42:00+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/dag1_kafka_processing.py', '--cfg-path', '/tmp/tmp_m5aa88b']
[2025-10-28T23:43:01.234+0000] {standard_task_runner.py:88} INFO - Job 65: Subtask ConsumKafka
[2025-10-28T23:43:01.303+0000] {task_command.py:423} INFO - Running <TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-28T23:42:00+00:00 [running]> on host 9680281a20bc
[2025-10-28T23:43:01.405+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag1_calcul_trajet_kafka' AIRFLOW_CTX_TASK_ID='ConsumKafka' AIRFLOW_CTX_EXECUTION_DATE='2025-10-28T23:42:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-28T23:42:00+00:00'
[2025-10-28T23:43:01.411+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.5', 9092)]>: connecting to kafka:9092 [('172.18.0.5', 9092) IPv4]
[2025-10-28T23:43:01.416+0000] {conn.py:618} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <checking_api_versions_recv> [IPv4 ('172.18.0.5', 9092)]>: Broker version identified as 2.6
[2025-10-28T23:43:01.417+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Connection complete.
[2025-10-28T23:43:01.419+0000] {subscription_state.py:181} INFO - Updating subscribed topics to: ('source',)
[2025-10-28T23:43:01.420+0000] {dag1_kafka_processing.py:71} INFO - Consommateur Kafka créé pour le topic 'source'. En attente de messages...
[2025-10-28T23:43:01.424+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.5', 9092)]>: connecting to kafka:9092 [('172.18.0.5', 9092) IPv4]
[2025-10-28T23:43:01.425+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Connection complete.
[2025-10-28T23:43:01.425+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Closing connection. 
[2025-10-28T23:43:01.529+0000] {cluster.py:401} INFO - Coordinator for group/***-consumers-dag1 is ('coordinator-1', 'kafka', 9092, None)
[2025-10-28T23:43:01.529+0000] {base.py:788} INFO - Discovered coordinator coordinator-1 for group ***-consumers-dag1
[2025-10-28T23:43:01.530+0000] {base.py:848} INFO - Starting new heartbeat thread
[2025-10-28T23:43:01.531+0000] {consumer.py:365} INFO - Revoking previously assigned partitions () for group ***-consumers-dag1
[2025-10-28T23:43:01.532+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: NodeNotReadyError: coordinator-1
[2025-10-28T23:43:01.534+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.5', 9092)]>: connecting to kafka:9092 [('172.18.0.5', 9092) IPv4]
[2025-10-28T23:43:01.535+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Connection complete.
[2025-10-28T23:43:01.636+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-28T23:43:01.641+0000] {base.py:637} INFO - Received member id kafka-python-2.2.15-7ff7289b-f8e2-4a4a-8f3a-0d42672620e7 for group ***-consumers-dag1; will retry join-group
[2025-10-28T23:43:01.642+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: [Error 79] MemberIdRequiredError
[2025-10-28T23:43:01.642+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-28T23:43:04.648+0000] {base.py:594} INFO - Successfully joined group ***-consumers-dag1 <Generation 1 (member_id: kafka-python-2.2.15-7ff7289b-f8e2-4a4a-8f3a-0d42672620e7, protocol: range)>
[2025-10-28T23:43:04.649+0000] {base.py:596} INFO - Elected group leader -- performing partition assignments using range
[2025-10-28T23:43:04.656+0000] {subscription_state.py:254} INFO - Updated partition assignment: [('source', 0)]
[2025-10-28T23:43:04.658+0000] {consumer.py:249} INFO - Setting newly assigned partitions (('source', 0),) for group ***-consumers-dag1
[2025-10-28T23:43:04.661+0000] {fetcher.py:418} INFO - Resetting offset for partition ('source', 0) to offset 0.
[2025-10-28T23:43:11.429+0000] {base.py:1052} INFO - Stopping heartbeat thread
[2025-10-28T23:43:11.431+0000] {base.py:891} INFO - Leaving consumer group (***-consumers-dag1).
[2025-10-28T23:43:11.439+0000] {base.py:906} INFO - LeaveGroup request for group ***-consumers-dag1 returned successfully
[2025-10-28T23:43:11.441+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Closing connection. 
[2025-10-28T23:43:11.441+0000] {fetcher.py:786} INFO - Fetch to node 1 failed: Cancelled: <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>
[2025-10-28T23:43:11.445+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Closing connection. 
[2025-10-28T23:43:11.447+0000] {dag1_kafka_processing.py:77} WARNING - Aucun message reçu du topic 'source'. Le DAG s'arrêtera.
[2025-10-28T23:43:11.452+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-10-28T23:43:11.472+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag1_calcul_trajet_kafka, task_id=ConsumKafka, execution_date=20251028T234200, start_date=20251028T234301, end_date=20251028T234311
[2025-10-28T23:43:11.512+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-28T23:43:11.546+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
